app/views/exam_templates/view_logs.html.erb

```diff
@@ -1,5 +1,11 @@
 <% content_for :title, "#{@assignment.short_identifier} #{SplitPdfLog.model_name.human.pluralize}" %>
 
+<% if flash[:split] %>
```

Comment:
This is what I meant by "good basic idea, but use `session` instead of `flash`."